java/se/sics/mspsim/util/ConfigManager.java

```diff
@@ -58,11 +58,7 @@ public class ConfigManager {
   protected Properties properties = null;
 
   public ConfigManager() {
-    this(null);
-  }
-
-  public ConfigManager(ConfigManager parent) {
-    this.parent = parent;
+    this.parent = null;
```

Comment:
This essentially disables support for using parents, and if so, the parent variable and handling code should also be removed?

Reply:
Fixed.